test/SlowTests/Issues/RavenDB-19967.cs

```diff
@@ -358,7 +358,7 @@ public async Task TombstoneCleaningAfterEtlLoaderDisabled(EtlType etlType)
                         blockerType = ITombstoneAware.TombstoneDeletionBlockerType.QueueEtl;
                         break;
                     case EtlType.Snowflake:
-                        var snowflakeConnectionString = new SnowflakeConnectionString { Name = store.Identifier, ConnectionString = global::Tests.Infrastructure.ConnectionString.SnowflakeConnectionString.Instance.VerifiedConnectionString.Value};
+                        var snowflakeConnectionString = new SnowflakeConnectionString { Name = store.Identifier,  ConnectionString = "ACCOUNT = SnowflakeAccount; USER = SnowflakeUser; PASSWORD = SnowflakePassword",};
```

Comment:
?

Reply:
That just need _a_ value, and otherwise would fail because I don't have the right env variable. That isn't needed for actual testing.